tests/ci/report.py

```diff
@@ -1,8 +1,19 @@
 # -*- coding: utf-8 -*-
 from ast import literal_eval
 from dataclasses import dataclass
+import dataclasses
```

Comment:
```suggestion
from dataclasses import asdict, dataclass
```

and according changes in line 317

Reply:
done